addons/website_quote/views/website_quotation.xml

```diff
@@ -148,7 +151,7 @@
                                   </a>
                               </div>
                           </div>
-                          <div class="text-center" t-if="need_payment" style="padding: 10px">
+                          <div class="text-center" t-if="need_payment or (quotation.state in ('sale') and quotation.require_payment and quotation.invoice_status != 'invoiced' and tx_state != 'done')" style="padding: 10px">
```

Comment:
In the case of up selling it is already showing the 'Pay Now' button , it will only be hidden in case the SO is fully invoiced...In case SO is changed and some line(s) is/are added then again the 'Pay Now' button will be visible..is this not ok ?? or I am misunderstanding anything?

Thanks